client/src/css/Alternative.css

```diff
@@ -1,43 +1,45 @@
 .Alternative {
-  margin: 1rem 0;
-  background: #6b71a5;
-  color: #ffffff;
-  font-size: 1.8rem;
+  margin: 0.5rem 0;
+  background-color: #6B71A5;
+  color: #FFF;
   display: flex;
-  box-shadow: 0px -2px 0px 0px rgba(0, 0, 0, 0.25) inset,
-              0px 2px 1px 0px rgba(0, 0, 0, 0.2);
+  box-shadow: 0 -0.125rem 0 0 rgba(0, 0, 0, 0.25) inset,
```

Comment:
Fine by me, just seems sort of odd to make an exception on this one case just because people can't be bothered. Will fix regardless, I do find it easier to just write 1-2px, but if you're using rem for whatever benefits it hold, why make exceptions, you know?